vmdb/app/models/miq_host_provision_workflow.rb

```diff
@@ -75,12 +75,15 @@ def allowed_hosts(options={})
   end
 
   def allowed_ws_hosts(options={})
-    Host.find(:all, :conditions=>["mac_address is not NULL"]).find_all {|h| h.ipmi_enabled}
+    Host.where("mac_address is not NULL").select { |h| h.ipmi_enabled }
```

Comment:
Note, ipmi_enabled is an alias of ipmi_config_valid? which is a method with logic basic on the ipmi_address column, ipmi_enabled is not a column...